src/hubverse_infrastructure/hubs/hubs.yaml

```diff
@@ -5,5 +5,8 @@ hubs:
 - hub: hubverse-cloud
   org: Infectious-Disease-Modeling-Hubs
   repo: hubverse-cloud
+- hub: bsweger-flusight-forecast
+  org: bsweger
```

Comment:
I actually don't know if this will work for a personal GitHub account (instead of an org). We'll find out!